python/huggingfaceserver/huggingfaceserver/vllm/vllm_completions.py

```diff
@@ -402,16 +404,16 @@ def _create_logprobs(
             step_top_logprobs = top_logprobs[i]
             if step_top_logprobs is not None:
                 token_logprob = step_top_logprobs[token_id].logprob
+                token = step_top_logprobs[token_id].decoded_token
```

Comment:
So the original logic assumes that `decoded_token` still exists when `logprob` is turned off?

Reply:
Actually, it is trying to access step_top_logprobs when it is None.